hack/copy-frontend-src-to-ossmc.sh

```diff
@@ -34,6 +34,7 @@ SOURCE_REPO_URL="${DEFAULT_SOURCE_REPO_URL}"
 # The git ref (branch or tag name) to checkout when cloning the source repo
 DEFAULT_SOURCE_REF="master"
 SOURCE_REF="${DEFAULT_SOURCE_REF}"
+GIT_REF="${SOURCE_REF}"
```

Comment:
This should move down after the options are processed (i.e. below line 85). Otherwise, GIT_REF will always be the ${DEFAULT_SOURCE_REF} when `-pr` is not used. You want GIT_REF to pick up SOURCE_REF that might be changed when someone passes in `-sr`.